solver/llbsolver/solver.go

```diff
@@ -340,6 +340,13 @@ func inlineCache(ctx context.Context, e remotecache.Exporter, res solver.CachedR
 	if !ok {
 		return nil, nil
 	}
+
+	if compressionopt.Type == compression.Nydus {
+		// Since the nydus image layer does not correspond to the cache records,
+		// the inline cache export of nydus image is disabled here.
+		return nil, errors.Errorf("inline cache with %s compression is unsupported", compressionopt.Type)
+	}
```

Comment:
Is the same limitation applied to non-inline remote cache as well?

Reply:
Thanks for pointing it out, added a check for this. Since the nydus compression type does not implement decompression at the moment, it does not support remotecache exports as a compression format.